dbms/src/Debug/MockComputeServerManager.cpp

```diff
@@ -52,6 +55,22 @@ void MockComputeServerManager::startServers(const LoggerPtr & log_ptr, Context &
     prepareMockMPPServerInfo();
 }
 
+void MockComputeServerManager::startServers(const LoggerPtr & log_ptr, int start_idx)
+{
+    for (const auto & server_config : server_config_map)
+    {
+        TiFlashSecurityConfig security_config;
+        TiFlashRaftConfig raft_config;
+        raft_config.flash_server_addr = server_config.second.addr;
+        Poco::AutoPtr<Poco::Util::LayeredConfiguration> config = new Poco::Util::LayeredConfiguration;
```

Comment:
Do we need to set proper gtest-only config values for some resource related configs, like max_streams, scheduler's soft/hard thread limit and so on laterly?

Reply:
> Do we need to set proper gtest-only config values for some resource related configs, like max_streams, scheduler's soft/hard thread limit and so on laterly?

Good idea, we need to summarize all the config related to mpp test, then wrap them up to have a precise control of how the tests be executed.